Makefile

```diff
@@ -179,28 +178,37 @@ build: validate-go-version go.sum
 build-linux: go.sum
 	WITH_LEDGER=false GOOS=linux GOARCH=amd64 $(MAKE) build
 
+DENOM ?= nhash
+MIN_FLOOR_PRICE ?= 1905nhash
+
```

Comment:
Might be useful to also have chain-id configurable.

Reply:
good idea added `CHAIN_ID ?= testing`